docs/source/dev_guide/releasing-e3sm-diags.rst

```diff
@@ -128,68 +52,31 @@ Remember to write a description.
 
     An example of a completed page to release the code
 
-3. Click "Publish release".
-
-Updating The sha256
```

Comment:
No more SHA256 for `conda/meta.yaml` since we are using now using source from git (`git_rev`), rather than tarball.